### DISCUSSION TITLE
Using SLEAP with an AMD GPU

### DISCUSSION THREAD

**Discussion by ltsully55** [talmolab/sleap#1641] [Help!]
I know that tensorflow is an NVIDIA specific system. I am wondering if there any way to integrate an AMD GPU (Recently upgraded from a 1660 Super to a Ryzen 6800xt), into the work flow. Any suggestions are helpful. If all else fails I can sort out the remote access!

**Comment by roomrys** [comment]
Hi @ltsully55,

We do not currently plan to support AMD GPUs, but we would be open to exploring that option once our new neural network backend is ready (~ few months).

Thanks,
Liezl
